metrics/dropwizard-metrics/src/main/java/org/apache/iotdb/metrics/dropwizard/reporter/DropwizardJmxReporter.java

```diff
@@ -47,18 +47,26 @@ public boolean start() {
               .build();
       jmxReporter.start();
     } catch (Exception e) {
-      LOGGER.error("Failed to start Dropwizard JmxReporter, because {}", e.getMessage());
+      jmxReporter = null;
+      LOGGER.warn("Dropwizard JmxReporter failed to start, because ", e);
       return false;
     }
+    LOGGER.info("Dropwizard JmxReporter start!");
     return true;
   }
 
   @Override
   public boolean stop() {
-    if (jmxReporter != null) {
-      jmxReporter.stop();
-      jmxReporter = null;
+    try {
+      if (jmxReporter != null) {
+        jmxReporter.stop();
+        jmxReporter = null;
+      }
+    } catch (Exception e) {
```

Comment:
Which exception will be catched here?

Reply:
To catch unexpected exception throw by jmxReporter stop method